Assets/Project/Scripts/Editor/EditorExtension.cs

```diff
@@ -18,7 +18,19 @@ public static class EditorExtension
             if (!property.isArray) return;
 
             property.isExpanded = EditorGUILayout.Foldout(property.isExpanded, property.displayName);
+            EditorGUILayout.BeginHorizontal();
             EditorGUILayout.PropertyField(property.FindPropertyRelative("Array.size"));
+            if (GUILayout.Button("-", GUILayout.Width(30))) {
+                if (property.arraySize > 0 && EditorUtility.DisplayDialog("削除確認", "まじで消すの？？復元できないぞ！？", "まじで消す", "やめとこう")) {
```

Comment:
```suggestion
                if (property.arraySize > 0 && !EditorUtility.DisplayDialog("削除確認", "まじで消すの？？復元できないぞ！？", "やめとこう", "まじで消す")) {
```
間違って消しそうだから、デフォルト値がNoになるようにこっちにしても良い？

Reply:
cb0616a